src/assets/main.scss

```diff
@@ -92,3 +92,7 @@ $danger: $navbar-background-color;
 #start-button:hover {
   background-color: darken($navbar-color-transparent, 5%);
 }
+
+#contact-embed {
+  min-height: 100vh;
```

Comment:
`100vh` sets the height of the element to 100% of the height of the viewport -- i.e., all of the space available to the browser window minus browser chrome.  As such, it is guaranteed to flow off the bottom of the page in this case.

In this case, using `height: 100%;` and setting:
```
.post {
  display: flex;
  flex-direction: column;
}

.post-content {
  flex-grow: 1;
}
```
yields a much better solution.

I also like `width: 100%;` here too -- while it won't affect the width of the form itself, it will center it in the page, and mean that mouse-wheeling in the empty space will scroll the form.  But that's just an opinion.

Reply:
(I'm also in the school of thought that says you never style `id`s, only ever `class`es, but that's not a convention observed in this project, and there's no need to start now.)

Reply:
Sure. Go ahead and push a commit with the changes.